timescaledb/how-to-guides/time-buckets/index.md

```diff
@@ -44,7 +46,7 @@ which is a Friday. It doesn't start on April 20, which is the immediately
 preceding Monday. It starts on April 13, because you can get to April 13, 2020,
 by counting in two-week increments from January 3, 2000.
 
-<!-- insert time bucket origin diagram -->
+<!-- TODO: insert time bucket origin diagram -->
```

Comment:
I think Vale looks for FIXME, should we add TODO?

Reply:
I'm fine either way! Personally I use TODO for "would be good to have this but if we don't get to it for months that'll be fine, shouldn't block merge" and FIXME for "this is crucial and should block merge."

So I learn towards not cluttering up Vale with TODO admonitions, but I will bow to group consensus :)

Reply:
nope, that makes perfect sense. thanks!